### PR TITLE
Trigger event on server flush.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
Fixes #309

**Comment by almet** [comment]
Thanks, that's useful. It also showcases that we're missing documentations for events, since no docs are to be updated yet :(.

File: kinto/tests/test_views_flush.py [hunk 1]
```diff
@@ -57,3 +83,9 @@ def test_removes_every_records_of_everykind(self):
         self.app.get(self.collection_url,
                      headers=self.alice_headers,
                      status=403)
+
+    def test_event_triggered_post(self):
+        self.app.get(self.collection_url, headers=self.headers)
+        self.app.get(self.collection_url, headers=self.alice_headers)
+        self.app.post('/__flush__', headers=self.headers, status=202)
+        self.assertEqual(len(self.events), 1)
```
Comment (almet) [inline kinto/tests/test_views_flush.py]: We probably should check here that the event is of the right type.

File: kinto/views/flush.py [hunk 2]
```diff
@@ -8,9 +8,16 @@
                 path='/__flush__')
 
 
+class ServerFlushed(object):
```
Comment (almet) [inline kinto/views/flush.py]: This should go in a `kinto.events` module (to be created)

Comment (johngian) [inline kinto/views/flush.py]: That was my initial take (following ``cliquet.events`` paradigm) but I wasn't sure if we needed to create that for just one event. I will move the changes to the new module.

**Comment by almet** [comment]
Thanks!

**Comment by leplatrem** [comment]
Excellent, I need this for my kinto-attachments plugin !

**Pull request merged**